lib/Doctrine/MongoDB/CursorInterface.php

```diff
@@ -253,4 +253,13 @@ public function tailable($tail = true);
      * @return self
      */
     public function timeout($ms);
+
+    /**
+     * Wrapper method for MongoCursor::maxTimeMS().
+     *
+     * @see http://php.net/manual/en/mongocursor.maxtimems.php
+     * @param integer $ms
+     * @return self
+     */
+    public function maxTimeMS($ms);
```

Comment:
@alcaeus: We may have shot ourselves in the foot here, but AFAIK we cannot add a new method to the interface without bumping to a new major version. This has come up with the collection interfaces in Doctrine Common before.

/cc @Ocramius